packages/shell-api/src/collection.ts

```diff
@@ -164,7 +164,7 @@ export default class Collection extends ShellApiWithMongoClass {
   async aggregate(
     pipeline: Document[],
     options: Document & { explain: ExplainVerbosityLike }
-  ): Promise<Document>;
+  ): Promise<AggregationCursor>;
```

Comment:
But the existing typings here are correct? The return type in this overload is an explain document, not a cursor